spec/examples/complete/seed.manifest.json

```diff
@@ -61,21 +74,34 @@
           "name": "MOUNT_PATH",
           "path": "/the/container/path",
           "mode": "ro"
+        },
+        {
+          "name": "WRITE_PATH",
+          "path": "/write",
+          "mode": "rw"
         }
       ],
       "settings": [
+        {
+          "name": "VERSION",
+          "secret": false
+        },
         {
           "name": "DB_HOST",
           "secret": false
+        },
+        {
+          "name": "DB_PASS",
+          "secret": true
         }
       ]
     },
     "resources": {
       "scalar": [
-        { "name": "cpu", "value": 1.0 },
-        { "name": "mem", "value": 128.0 },
-        { "name": "sharedMem", "value": 0.0 },
-        { "name": "disk", "value": 10.0, "inputMultiplier": 4.0 }
+        { "name": "cpus", "value": 10.0 },
```

Comment:
I'm not sure we want a primary example to require such high resources. This could be prohibitive for some machines. I'd say we shouldn't go above 2 CPUs and 512MB memory. Disk isn't a huge deal but we don't need anything large so I see no reason to do it. Feel free to correct me if I've missed something.